framework/src/base/Moose.C

```diff
@@ -280,6 +281,7 @@ addActionTypes(Syntax & syntax)
                            "(create_problem_custom)"
                            "(create_problem_default)"
                            "(create_problem_complete)"
+                           "(check_generated_mesh)"
```

Comment:
With its new `post_mesh_prepared` name, let's move this to right after `setup_mesh_complete`

Reply:
Please see my last commit. CIVET was crashing without that last revert commit.

Reply:
I think we need to understand why it was crashing. If the task relies on something more than the mesh being prepared, then we need to rename it

Reply:
Oh I see the issue. You're calling `_problem->mesh()`. In an action you should already have access to `_mesh` and `_displaced_mesh` members. You don't need to go through the `_problem`

Reply:
Ah, that makes sense. Will make that change.